src/Middleware/SpaServices.Extensions/src/Util/TcpPortFinder.cs

```diff
@@ -21,5 +22,19 @@ public static int FindAvailablePort()
                 listener.Stop();
             }
         }
+
+        public static bool TestPortAvailability(int port)
```

Comment:
To me this naming makes it ambiguous what a `true` result means. Maybe `IsPortAvailable`?